src/pages/AdminPage/AdminPage.tsx

```diff
@@ -81,17 +78,8 @@ const AdminPage = () => {
           <Text h2 css={{ my: '$xl' }}>
             {selectedCircle?.name}
           </Text>
-          {!isMobile ? (
+          {!isMobile && (
             <Flex css={{ flexGrow: 1, justifyContent: 'flex-end', gap: '$md' }}>
-              <Button
```

Comment:
related to comment above: keep this button here, but make it a link that goes to the new page. (You can do that [like this](https://github.com/coordinape/coordinape/blob/claims/src/pages/ClaimsPage/ClaimsNavButton.tsx#L26))

Reply:
Done